openstack/blockstorage/extensions/quotasets/results.go

```diff
@@ -33,6 +33,66 @@ type QuotaSet struct {
 	BackupGigabytes int `json:"backup_gigabytes"`
 }
 
+// QuotaDetailSet represents details of both operational limits of block
+// storage resources and the current usage of those resources.
+type QuotaDetailSet struct {
```

Comment:
Similar to the `GetUsage` function name, I think this should be called `QuotaSetUsage` since it's returning a set of quota items and their usage.